webapp/carts/models.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from django.db import models
+
+
+User = settings.AUTH_USER_MODEL
+
+
+class Cart(models.Model):
+    #allowed nulls and blanks for future guest checkout
+    user = models.ForeignKey(
+        User,
+        null=True,
+        blank=True)
+
+    book = models.ForeignKey(
+        'book_details.Book',
+        related_name='product')
+
+    total_price = models.DecimalField(
+        'total price of all items',
+        default=0.00,
+        max_digits=100,
+        decimal_places=2)
+
+    updated = models.DateTimeField(
+        auto_now=True)
+
+    timestamp = models.DateTimeField(
+        auto_now_add=True)
+
+    quantity = models.IntegerField(
+        'quanitity of books',
+        default=1)
+
+    book_price_quantity = models.DecimalField(
```

Comment:
What does is this field for?
The description given is not very clear.

Reply:
The total cost of that item units*price this will be moved over to the order model once implemented the idea is to give the best seller functionality later on.